tests/integration/cli/Cargo.toml

```diff
@@ -7,6 +7,9 @@ repository = "https://github.com/wasmerio/wasmer"
 edition = "2018"
 publish = false
 
+[dev-dependencies]
+rand = "0.8.5"
+
```

Comment:
[`tempfile`](https://crates.io/crates/tempfile) is useful for this workflow. (Don't rewrite your PR for this minor thing, it's just a suggestion for future tests)